base/src/main/java/smile/util/Paths.java

```diff
@@ -29,15 +30,15 @@
  */
 public interface Paths {
     /** Smile home directory. */
-    String home = System.getProperty("smile.home", "shell/src/universal/data");
```

Comment:
This line should not be changed.

Reply:
> This line should not be changed.

That line is not included in the final changes contained in this PR
![image](https://user-images.githubusercontent.com/1487283/149607237-8f405826-298d-4749-8495-edf32a120323.png)